lib/ansible/modules/network/eos/eos_eapi.py

```diff
@@ -187,44 +187,51 @@
 from ansible.module_utils.six import iteritems
 from ansible.module_utils.eos import eos_argument_spec, check_args
 
+
 def check_transport(module):
     transport = module.params['transport']
     provider_transport = (module.params['provider'] or {}).get('transport')
 
     if 'eapi' in (transport, provider_transport):
         module.fail_json(msg='eos_eapi module is only supported over cli transport')
 
+
 def validate_http_port(value, module):
     if not 1 <= value <= 65535:
         module.fail_json(msg='http_port must be between 1 and 65535')
 
+
 def validate_https_port(value, module):
     if not 1 <= value <= 65535:
         module.fail_json(msg='http_port must be between 1 and 65535')
 
+
 def validate_local_http_port(value, module):
     if not 1 <= value <= 65535:
         module.fail_json(msg='http_port must be between 1 and 65535')
 
+
 def validate_vrf(value, module):
     out = run_commands(module, ['show vrf'])
     configured_vrfs = re.findall('^\s+(\w+)(?=\s)', out[0], re.M)
     configured_vrfs.append('default')
     if value not in configured_vrfs:
         module.fail_json(msg='vrf `%s` is not configured on the system' % value)
 
+
 def map_obj_to_commands(updates, module, warnings):
     commands = list()
     want, have = updates
 
-    needs_update = lambda x: want.get(x) is not None and (want.get(x) != have.get(x))
```

Comment:
I think `def` statements are more readable and simple to understand. Also, using `def` will make code uniform. I do, agree lambda is prettier but I would use it where it is not assigned to any variable. e.g.,

`sorted(cars, key=lambda car: car.color)`